react/features/facial-recognition/actions.js

```diff
@@ -118,8 +113,12 @@ export function loadWorker() {
             }
         };
         worker.postMessage({
-            id: 'SET_MODELS_URL',
-            url: baseUrl
+            type: INIT_WORKER,
+            url: baseUrl,
+            windowScreenSize: window.screen ? {
```

Comment:
this won't work on mobile will it?

Reply:
Just tested it on mobile web, it works.

Reply:
And about native mobile, it is not intended to work

Reply:
If there is no screen size sent the worker has some default screen size.

Reply:
But I think it would work to send the screen size on mobile, the screen property for window having full support https://developer.mozilla.org/en-US/docs/Web/API/Window/screen